onnxruntime/contrib_ops/cuda/fused_conv.cc

```diff
@@ -80,12 +82,13 @@
                                                     Base::s_.y_tensor,
                                                     Base::s_.y_data));
       if (has_b) {
-        CUDNN_RETURN_IF_ERROR(cudnnAddTensor(Base::CudnnHandle(), &alpha, Base::s_.b_tensor, Base::s_.b_data,
-                                             &alpha, Base::s_.y_tensor, Base::s_.y_data));
+        //Base::s_.b_tensor
+        has_b = cudnnStatus_t::CUDNN_STATUS_SUCCESS != cudnnAddTensor(Base::CudnnHandle(), &alpha, Base::s_.b_tensor, Base::s_.b_data,
```

Comment:
Not clear why we are swallowing this error here and below. I thought this is supposed to be fixed.